main.go

```diff
@@ -17,10 +17,6 @@ var VERSION string
 
 // The main entrypoint for Terragrunt
 func main() {
-	// Log the terragrunt version in debug mode. This helps with debugging issues and ensuring a specific version of
-	// terragrunt used.
-	util.Debugf(util.CreateLogger(""), "Terragrunt Version: %s", VERSION)
```

Comment:
Looks like this used to log the Terragrunt version in debug mode during boot. We shouldn't lose that behavior.

Reply:
Is this still missing?

Reply:
I initially moved it to later stage (because we had no logger here). Now with global logger it makes sense to move it back.